### PR TITLE
Add CloudSQL and SAP HANA to database options

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: wrangler-proto/src/main/java/co/cask/wrangler/proto/db/AllowedDriverInfo.java [hunk 1]
```diff
@@ -29,12 +29,14 @@
   private final String name;
   @SerializedName("default.port")
   private final String port;
+  private final boolean basicAllowed;
 
-  public AllowedDriverInfo(String clazz, String label, String tag, String name, String port) {
+  public AllowedDriverInfo(String clazz, String label, String tag, String name, String port, boolean basicAllowed) {
```
Comment (bdmogal) [inline wrangler-proto/src/main/java/co/cask/wrangler/proto/db/AllowedDriverInfo.java:35]: so we know in future, can you add a comment as to why we maintain two classes for what seem like similar things?

File: wrangler-service/src/main/java/co/cask/wrangler/service/database/DatabaseHandler.java [hunk 2]
```diff
@@ -166,6 +168,11 @@ public String getTag() {
     public String getPort() {
       return port;
     }
+
+    public boolean getBasicAllowed() {
```
Comment (bdmogal) [inline wrangler-service/src/main/java/co/cask/wrangler/service/database/DatabaseHandler.java]: should be ``isBasicAllowed`` per convention

**Review by bdmogal — COMMENTED** [review]
couple comments ...

File: wrangler-proto/src/main/java/co/cask/wrangler/proto/db/AllowedDriverInfo.java (hunk 1 above)
Comment (elfenheart) [reply wrangler-proto/src/main/java/co/cask/wrangler/proto/db/AllowedDriverInfo.java:35]: I'm quite unsure why there are two classes for this

**Review by bdmogal — APPROVED** [review]

**Pull request merged**